community/kernel/src/main/java/org/neo4j/kernel/api/labelscan/LabelScanStore.java

```diff
@@ -91,8 +92,22 @@ public void rebuilt( long roughNodeCount )
 
     /**
      * Acquire a writer for updating the store.
+     *
+     * @return {@link LabelScanWriter} which can modify the {@link LabelScanStore}.
+     */
+    default LabelScanWriter newWriter()
```

Comment:
As above comment mentions, this has now been removed and so it looks like it used to do, with only a `newWriter()` method.